app/code/community/Bolt/Boltpay/controllers/OrderController.php

```diff
@@ -170,6 +168,9 @@ public function firecheckoutcreateAction()
                 }
             }
 
+            $checkout->applyShippingMethod($this->getRequest()->getPost('shipping_method', false));
+            $quote->getShippingAddress()->setShippingMethod($this->getRequest()->getPost('shipping_method', false));
```

Comment:
Does exist a difference between these two methods?

`$checkout->applyShippingMethod(...)`
and
`$quote->getShippingAddress()->setShippingMethod(...)`

It seems that they should apply the same thing, but I do not have the Firecheckout module right now.

Reply:
@YevhenShBolt 
Yes.  There is a difference.

Reply:
It seemed to me that these both methods add the same data to the same quote.

Reply:
@YevhenShBolt I've made an addition pull request to address this issue.

@daisy1754 Please note that https://github.com/BoltApp/bolt-magento1/pull/151, once approved, should be merged to this PR first before this is merged to develop.